src/app/utils/ValidatorUtils.ts

```diff
@@ -370,13 +364,28 @@ export class ValidatorUtils extends CacheModule {
 		}
 
 		const result = request.parse(response)[0]
-		this.lastFreshTime = Date.now()
 		this.currentEpoch = result.currentEpoch[0]
 		this.olderEpoch = result.olderEpoch[0]
 		this.rocketpoolStats = result.rocketpool_network_stats[0]
 		const validatorEffectivenessResponse = result.effectiveness
 		const validatorsResponse = result.validators
 
+		this.syncCommitteesStats = null
+		const hideSyncCommitteesStats = true // wait for the explorer to show sync committee stats too before showing them in the app
```

Comment:
I don't think this is necessary. We can draft the app release once it's live on the explorer

Reply:
As discussed, we will not delay the sync committee luck on the app. I have removed the `hideSyncCommitteesStats` special handling.